Update packages README (#69)
signalwire/signalwire-node: packages/react-native/README.md

npm install @signalwire/react-native
```

## WebRTC Engine

Our package `@signalwire/react-native` depends on [react-native-webrtc](https://github.com/react-native-webrtc/react-native-webrtc) and will try to install it automatically for you in the `postinstall` script.\

> It could happen that something goes wrong during the linking process of the native libraries. If your app does not compile you must follow these steps to troubleshoot:

Reviewer: ```suggestion
It's possible that an error will occur during the linking process of the native libraries. If your app does not compile, follow these steps to troubleshoot:
```

Then instantiate the client:
> Note: host / project / token are required

> Make sure to subscribe to the `signalwire.notification` of type `refreshToken` to know when the client needs to refresh you JWT to keep the session live!\

Reviewer: ```suggestion
Make sure to subscribe to the `signalwire.notification` of type `refreshToken` to know when the client needs to refresh you JWT to keep the session live!
```
